urbackupserver/www2/src/api/urbackupserver.ts

```diff
@@ -297,7 +297,7 @@ class UrBackupServer {
   };
 
   // Start a backup of type `backupType` for client with id `clientId`
-  startBackup = async (clientId: [ClientIdType], backupType: BackupType) => {
+  startBackup = async (clientId: ClientIdType[], backupType: BackupType) => {
```

Comment:
Meant to have an array type here anyway. Guess I never tested it with multiple clients...